package.json

```diff
@@ -20,10 +20,11 @@
     "changelog:validate": "yarn workspaces foreach --all --no-private --parallel --interlaced --verbose run changelog:validate",
     "create-package": "ts-node scripts/create-package",
     "lint": "yarn lint:eslint && yarn lint:misc --check && yarn constraints && yarn lint:dependencies",
+    "lint:clear-cache": "rimraf .eslintcache",
     "lint:dependencies": "depcheck && yarn dedupe --check",
     "lint:dependencies:fix": "depcheck && yarn dedupe",
-    "lint:eslint": "eslint . --cache --ext js,cjs,mjs,ts",
-    "lint:fix": "yarn lint:eslint --fix && yarn lint:misc --write && yarn constraints --fix && yarn lint:dependencies:fix",
+    "lint:eslint": "eslint . --cache --ext js,cjs,mjs,ts,tsx",
```

Comment:
Added `.tsx` extension to ESLint configuration to ensure React TypeScript components are properly linted. This helps maintain consistent code quality across both JavaScript and TypeScript React and React Native files in our codebase.